libse/Utilities.cs

```diff
@@ -499,7 +515,10 @@ public static string AutoBreakLinePrivate(string text, int maximumLength, int me
             }
 
             string s = RemoveLineBreaks(text);
-            if (HtmlUtil.RemoveHtmlTags(s, true).Length < mergeLinesShorterThan)

```

Comment:
By doing this, if you pass a single line dialog which its length is < mergedLinesShorterThan, it will be skipped